oss-fuzz/build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -eu
+export CLANG_PATH=/usr/local/bin
+export ASAN_SYMBOLIZER_PATH=/usr/local/bin/llvm-symbolizer
+export PACKAGES_PATH="$SRC/hbfa-fl/HBFA:$SRC/edk2"
+export EDK_TOOLS_PATH="$SRC/edk2/BaseTools"
+export CONF_PATH="$SRC/edk2/Conf"
+export PYTHON_COMMAND=/usr/local/bin/python
```

Comment:
Is this correct? Wouldn't it be better to have something like "export PYTHON_COMMAND=`which python`"? And what uses this PYTHON_COMMAND env?

Reply:
I did an oss-fuzz build with this removed and it worked fine, so this is just an artifact from earlier version that's no longer needed and can be removed.

Reply:
Agreed, good catches; the env variable should be good to remove. Removing that and updating license info in header with SPDX line.